src/contrib/mlir/compiler/compiler.cpp

```diff
@@ -93,27 +93,27 @@ static llvm::cl::opt<bool> clPrintIRAfterAll(
 // *** Optimization flags ***
```

Comment:
I would change `print-ngraph-ir-after-all` to be `ngraph-print-after-all` to be consistent. There is also the other flags you recently added to dump obj file.

Reply:
Good point, thanks. Changing it to `ngraph-print-ir-after-all` for consistency with MLIR counterpart